views/login.hbs

```diff
@@ -33,25 +33,35 @@
         <!-- NAVBAR -->
         {{> navbar }}
 
-        <div class="houseline"></div>
-
-        <div class="background theme-yellow col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-xs-10 col-xs-offset-1">
+        <div class="background col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-xs-12">
+            <h1>Log In</h1>
+            <h2>Enter email and password</h2>
             <form class='ideahacks-form'>
-                <p class='error-message'>{{ gError }}</p>
+                <div class="input-wrapper">
+                    <p class='error-message'>{{ gError }}</p>
+
+                    {{!-- Email --}}
+                    <div>
+                        <label for="username">Email</label>
+                        <input type="text" class="form-control" name="username" id="username" placeholder="Email" required>
+                    </div>
 
-                <label for="username">Email</label>
-                <input type="text" class="form-control" name="username" id="username" placeholder="Email" required>
+                    {{!-- Password --}}
+                    <div>
+                        <label for="password">Password</label>
+                        <input type="password" class="form-control" name="password" id="password" placeholder="Password" required>
+                    </div>
 
-                <label for="password">Password</label>
-                <input type="password" class="form-control" name="password" id="password" placeholder="Password" required>
+                    <p class="warning">@g.ucla.edu emails only!</p>
+                </div>
 
-                <button type="submit" class="login">LOG IN</button>
+                <button type="submit" class="login">Log In</button>
                 <br>
-                <p>Don't have an account? <a href='/registration'>Sign up!</a></p>
+                <p class="no-account">Don't have an account? <a href='/registration'>Register!</a></p>
                 <p><a name='password-recover'>Forgot your password?</a></p>
+
                 <hr>
                 <a href="/login/google"><img src="/img/login/google.png" id="google" draggable="false"></a>
```

Comment:
Like I mentioned in the above unresolved comment, can you move the `hr` and button up to the main login section as well? Sorry I wasn't specific before; I always forget GitHub doesn't show exactly which lines you comment on